libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -41,14 +44,15 @@ typedef struct OVModel{
     ie_blob_t *input_blob;
 } OVModel;
 
-static const AVClass dnn_openvino_class = {
-    .class_name = "dnn_openvino",
-    .item_name  = av_default_item_name,
-    .option     = NULL,
-    .version    = LIBAVUTIL_VERSION_INT,
-    .category   = AV_CLASS_CATEGORY_FILTER,
+#define OFFSET(x) offsetof(OVContext, x)
+#define FLAGS AV_OPT_FLAG_FILTERING_PARAM | AV_OPT_FLAG_VIDEO_PARAM
+static const AVOption dnn_openvino_options[] = {
+    { "device", "device to run model", OFFSET(device_type), AV_OPT_TYPE_STRING, { .str = NULL }, 0, 0, FLAGS },
```

Comment:
can we make .str="CPU"?  so we don't need line 219-222

Reply:
Well, I think we can make .str="CPU". Will modify in next version.